docs/DeveloperGuide.md

```diff
@@ -247,16 +247,40 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+Priorities: High (must have) - `***`, Medium (nice to have) - `**`, Low (unlikely to have) - `*`
+
+| `***`  | As a …​                                   | I want to …​                                                    | So that …​                                                                                   |
+|----------|----------------------------------------------|--------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| `***`  | User                                         | view all the possible commands that I can execute                  | I know what functionalities I can make use of when using the program.                        |
+| `***`  | User                                         | add orders for the day                                             | I can add new orders to my database                                                 |
+| `***`  | User                                         | delete orders for the day                                          | I can remove unnecessary orders from my database                                                
+| `***`  | User                                         | list all my orders for the day                                     | I can view what orders I have to fulfil for the day                                          |
+| `***`  | User                                         | retrieve data stored in the previous sessions                      | (obvious)                                                                            |
+| `***`  | User logging in after a long time            | delete all orders, reset the application                           | I can start on a clean slate since the previous orders are now meaningless to me             |
+| `**`    | Advanced user                                | set up shortcuts and shortforms                                    | I can reduce the time it takes to enter orders.                                              |
+| `**`    | User                                         | receive reminders for the orders due soon                          | I remember and complete them on time.                                                        |
+| `**`    | User who's at least slightly experienced     | delete multiple orders at one go                                   | I can track mass cancellations if they occur                                                 |
```

Comment:
Maybe instead of "track", "execute"? Or something along that line.. Because we aren't really tracking the deletion, just actually removing it from the database.

Reply:
Got it, changing it to "I can reflect mass cancellations in my database if they occur"